src/util_lib/db.rs

```diff
@@ -276,6 +276,18 @@ pub fn u64_to_sql(x: u64) -> Result<i64, Error> {
     Ok(x as i64)
 }
 
+pub fn u64_opt_to_sql(x: Option<u64>) -> Result<Option<i64>, Error> {
+    match x {
+        Some(num) => {
+            if num > (i64::MAX as u64) {
```

Comment:
@jcnelson should I instead be returning an `Option<u128>` from this function to get the full range of u64?

Reply:
No, this is fine.  This and `u64_to_sql` are helper methods to just cast the u64 to an i64 if it's small enough, and store it to sqlite (which only understands signed 64-bit numbers).  It looks like it's the case already, but just be sure that whatever you're using this for will always fit into an i64 for the foreseeable future.  I think you're only using this on `until_burn_ht`, which is a burnchain block height, right?  You're not using this for token balances, which really should be u128?